apps/twap/src/app/orderbook/order.ts

```diff
@@ -0,0 +1,126 @@
+import { Column, PrimaryColumn, Entity } from 'typeorm';
+import {
+  bigIntToString,
+  bufferToString,
+  stringToBigInt,
+  stringToBuffer,
+} from '../utils/transformers';
+
+@Entity({ name: 'orders' })
+export class Order {
+  @PrimaryColumn('bytea', {
+    name: 'uid',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  uid: string;
+
+  @Column('bytea', {
+    name: 'owner',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  owner: string;
+
+  @Column('timestamp with time zone', { name: 'creation_timestamp' })
+  creationTimestamp: Date;
+
+  @Column('bytea', {
+    name: 'sell_token',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  sellToken: string;
+
+  @Column('bytea', {
+    name: 'buy_token',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  buyToken: string;
+
+  @Column('numeric', { name: 'sell_amount' })
+  sellAmount: number;
+
+  @Column('numeric', { name: 'buy_amount' })
+  buyAmount: number;
+
+  @Column('bigint', {
+    name: 'valid_to',
+    transformer: { from: stringToBigInt, to: bigIntToString },
+  })
+  validTo: bigint;
```

Comment:
Is there any format we can use for timestamp?

Reply:
![image](https://github.com/cowprotocol/cow-web-services/assets/5172699/c5ffa93d-0c59-49e8-a4ff-e32607ba0a66)
Not defined as a timestamp, as opposed to `creation_timestamp`. We can later on add transformers to these, but as of right now it might be unnecessary.